src/map/map.cpp

```diff
@@ -1754,10 +1756,11 @@ int map_search_freecell(struct block_list *src, int16 m, int16 *x,int16 *y, int1
 				if (spawn >= 100) return 0; //Limit of retries reached.
```

Comment:
```suggestion
```
The battle config is already limited [0,100]

Reply:
This will break the feature. It's different because if you set it to 100 the monster should not spawn at all, whereas if the value is lower than 100, it should spawn anyway.